src/main/java/org/junitpioneer/jupiter/package-info.java

```diff
@@ -6,6 +6,7 @@
  *     <li>{@link org.junitpioneer.jupiter.ClearSystemProperty} and {@link org.junitpioneer.jupiter.SetSystemProperty}</li>
  *     <li>{@link org.junitpioneer.jupiter.ClearEnvironmentVariable} and {@link org.junitpioneer.jupiter.SetEnvironmentVariable}</li>
  *     <li>{@link org.junitpioneer.jupiter.DefaultLocale} and {@link org.junitpioneer.jupiter.DefaultTimeZone}</li>
+ *     <li>{@link org.junitpioneer.jupiter.Issue}</li>
```

Comment:
Nice, I always forget to update the `package-info.java`. Good work. :sweat_smile: